demos/smart_framing_demo/cpp_gapi/smart_framing_demo_gapi.hpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+#pragma once
+
+#include <gflags/gflags.h>
+#include <utils/default_flags.hpp>
+
+
+DEFINE_INPUT_FLAGS
+DEFINE_OUTPUT_FLAGS
+
+static const char help_message[] = "Print a usage message.";
```

Comment:
It is unused file now, isn't it?

Reply:
Graph code rearranged. Don't completely understand  "put graph after 221 line "